internal/sbi/processor/processor.go

```diff
@@ -0,0 +1,18 @@
+package processor
+
+import (
+	"github.com/free5gc/udr/internal/database"
+	"github.com/free5gc/udr/pkg/app"
+)
+
+type Processor struct {
+	app.UdrApp
+	database.DbConnector
+}
+
+func NewProcessor(udr app.UdrApp, dbInplement database.DbConnector) *Processor {
+	return &Processor{
+		UdrApp:      udr,
+		DbConnector: dbInplement,
```

Comment:
DbConnector: mongodb.NewMongoDbConnector(udr.Config().Configuration.Mongodb),

Reply:
There will be an import cycle when I use mongodb NewMongoDbConnector, because the method "PatchDataToDBAndNotify" uses a method in the processor

Reply:
I have an idea to remove the call to processor.PreHandleOnDataChangeNotify from mongodb.PatchDataToDBAndNotify and place it at a higher level, where processor.PreHandleOnDataChangeNotify is called

Reply:
Hi @ian-cs12-NYCU 
You can define `NewDbConnector(dbType string)` in `database` package, it should return the connector instance based on `dbType`.
After that, `processor` package wouldn't have relation to `mognodb` package anymore.